lockbox-ios/View/SettingsView.swift

```diff
@@ -167,6 +167,7 @@ class SwitchSettingCellConfiguration: SettingCellConfiguration {
 class CheckmarkSettingCellConfiguration: SettingCellConfiguration {
     var isChecked: Bool = false
     var valueWhenChecked: Any?
+    var enabled: Bool = true
```

Comment:
could this be a property on all settings cells? it's possible that we will want to disable / enable other settings in the future.